node/impl/full/eth.go

```diff
@@ -2123,7 +2125,7 @@ func (e *EthEventHandler) EthSubscribe(ctx context.Context, p jsonrpc.RawParams)
 			}
 		}
 
-		f, err := e.EventFilterManager.Install(ctx, -1, -1, cid.Undef, addresses, keysToKeysWithCodec(keys), true)
+		f, err := e.EventFilterManager.Install(ctx, -1, -1, cid.Undef, addresses, keysToKeysWithCodec(keys), false)
```

Comment:
I guess this is a bugfix too? https://geth.ethereum.org/docs/interacting-with-geth/rpc/pubsub#logs is the only explicit mention of this I can find when searching but it does clearly say that you should expect reverts on an `eth_subscribe`.

Reply:
Yes the fact that this was `true` surprised me too.